src/View/Admin/WooTab.php

```diff
@@ -171,13 +182,17 @@ public function register_hooks() {
 		add_filter( 'pre_option_' . self::COUNTRY_CODE_FIELD, [ $this, 'get_woo_country' ] );
 		add_filter( 'woocommerce_admin_settings_sanitize_option_' . self::PHONE_NUMBER_FIELD, [ $this, 'sanitize_phone_number' ] );
 		add_filter( "woocommerce_get_settings_{$this->id}", [ $this, 'maybe_add_connection_button' ] );
-		add_action( 'woocommerce_admin_field_cc_connection_button', [ $this, 'add_cc_connection_button' ] );
-		add_action( 'woocommerce_admin_field_cc_cta_button', [ $this, 'render_cta_button' ] );
+		// add_action( 'woocommerce_admin_field_cc_cta_button', [ $this, 'render_cta_button' ] );
+		// add_action( 'woocommerce_admin_field_cc_connection_button', [ $this, 'add_go_back_button' ] );
 
```

Comment:
Same with these two. If we're for sure getting rid of these callbacks, then lets remove from the code completely.